Magento_GiftRegistry/templates/edit/registry.phtml

```diff
@@ -1,9 +1,9 @@
 <input type="hidden" name="entity_id" id="entity_id" value="<?= /* @escapeNotVerified */ $block->getEntity()->getEntityId(); ?>" />
 <input type="hidden" name="type_id" id="type_id" value="<?= /* @escapeNotVerified */ $block->getTypeId(); ?>" />
```

Comment:
long lines

Reply:
I think it's better to leave this 2 lines long b/c when it's formatted they looking strange. And for more, it's hidden inputs.